src/styles/components/_chat.scss

```diff
@@ -349,6 +362,15 @@
         width: 12px;
       }
 
+      &:after {
+        font-family: FontAwesome;
```

Comment:
Properties should be ordered float, font-family, left, margin-left, text-align, width